src/util/transifex-upload.js

```diff
@@ -0,0 +1,79 @@
+const apiBaseUrl = 'https://www.transifex.com/api/2/project';
```

Comment:
all of this code should only be happening in trusted dev environments (i.e. those we have given a key too)

this is potentially an argument to keeping the model of PR'ing the translation file and having the push happen on build, because it would allow all developers to add translations without us needing to give keys out

Reply:
(and code should not ship in production bundle)

Reply:
I'm not aware of a way to exclude the code specifically, but I can look into a way of loading the key dynamically instead of hardcoding.